app/neo/editors/database/DatabaseImportWizard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ClientContextProvider, ImportWizard } from '@axonivy/database-editor';
+import { Button, ThemeProvider } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useQueryClient } from '@tanstack/react-query';
+import i18next from 'i18next';
+import { I18nextProvider, useTranslation } from 'react-i18next';
+import { useDataClassesApi } from '~/data/data-class-api';
+import { useSortedProjects } from '~/data/project-api';
+import { useWorkspace } from '~/data/workspace-api';
+import { DatabaseClientNeo } from '~/neo/editors/database/database-client';
+import { useWebSocket } from '~/neo/editors/useWebSocket';
+
+export const DatabaseImportWizard = () => {
+  const ws = useWorkspace();
+  const app = ws?.baseUrl.substring(2) ?? 'invalid';
```

Comment:
I think `const { app } = useParams();` should work here

Reply:
nop, app name also contains the user, i.e. USER-WORKSPACENAME, because of uniqueness...